src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -103,3 +104,14 @@
 {% if swiss_qrbill %}
 <link rel="stylesheet" href="{% static "pretixplugins/banktransfer/swisscross.css" %}">
 {% endif %}
+
+<hr>
+<form class="form-inline helper-display-inline" method="post"
+        action="{% eventurl event "plugins:banktransfer:mail_invoice" order=order.code secret=order.secret %}">
+    {% csrf_token %}
+    <input type="email" name="email" class="form-control" value=""/>
```

Comment:
This input field could probably do with
- a `required` attribute
- a `<label>`, even if you hide it with `class="sr-only"`, to make screenreader users happy